Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -171,7 +171,10 @@ open class MessageContentCell: MessageCollectionViewCell {
     /// Positions the cell's `AvatarView`.
     /// - attributes: The `MessagesCollectionViewLayoutAttributes` for the cell.
     open func layoutAvatarView(with attributes: MessagesCollectionViewLayoutAttributes) {
-        guard attributes.avatarSize != .zero else { return }
+        guard attributes.avatarSize != .zero else {
```

Comment:
Actually, we want to just remove these lines completely. We still need to calculate the `X` and `Y` positions

Reply:
No early exit

Reply:
🤔 What's the reason? I think each subview did not depend on other's `frame`.

Reply:
Oops, I didn't look carefully.